src/layers/conv.jl

```diff
@@ -1,168 +1,99 @@
+end
+
+function conv_norm(kernel_size, ch::Pair{<:Integer, <:Integer}, outplanes,
+                   activation = identity; kwargs...)
+    inplanes, outplanes = ch
+    return conv_norm(kernel_size, inplanes, outplanes, activation; kwargs...)
 end
 
 """
-    depthwise_sep_conv_bn(kernelsize, inplanes, outplanes, activation = relu;
-                               rev = false, use_bn = (true, true),
-                               stride = 1, pad = 0, dilation = 1, [bias, weight, init],
-                               initβ = Flux.zeros32, initγ = Flux.ones32,
-                               ϵ = 1.0f-5, momentum = 1.0f-1)
+    depthwise_sep_conv_bn(kernel_size, inplanes, outplanes, activation = relu;
+                               prenorm = false, use_bn = (true, true),
+                               stride = 1, pad = 0, dilation = 1, [bias, weight, init])
 
 Create a depthwise separable convolution chain as used in MobileNetv1.
 This is sequence of layers:
 
-  - a `kernelsize` depthwise convolution from `inplanes => inplanes`
+  - a `kernel_size` depthwise convolution from `inplanes => inplanes`
   - a batch norm layer + `activation` (if `use_bn[1] == true`; otherwise `activation` is applied to the convolution output)
-  - a `kernelsize` convolution from `inplanes => outplanes`
+  - a `kernel_size` convolution from `inplanes => outplanes`
   - a batch norm layer + `activation` (if `use_bn[2] == true`; otherwise `activation` is applied to the convolution output)
 
 See Fig. 3 in [reference](https://arxiv.org/abs/1704.04861v1).
 
 # Arguments
 
-  - `kernelsize`: size of the convolution kernel (tuple)
+  - `kernel_size`: size of the convolution kernel (tuple)
   - `inplanes`: number of input feature maps
   - `outplanes`: number of output feature maps
   - `activation`: the activation function for the final layer
-  - `rev`: set to `true` to place the batch norm before the convolution
+  - `prenorm`: set to `true` to place the batch norm before the convolution
   - `use_bn`: a tuple of two booleans to specify whether to use batch normalization for the first and second convolution
   - `stride`: stride of the first convolution kernel
   - `pad`: padding of the first convolution kernel
   - `dilation`: dilation of the first convolution kernel
   - `bias`, `weight`, `init`: initialization for the convolution kernel (see [`Flux.Conv`](#))
-  - `initβ`, `initγ`: initialization for the batch norm (see [`Flux.BatchNorm`](#))
-  - `ϵ`, `momentum`: batch norm parameters (see [`Flux.BatchNorm`](#))
 """
-function depthwise_sep_conv_bn(kernelsize, inplanes, outplanes, activation = relu;
-                               rev = false, use_bn = (true, true),
-                               initβ = Flux.zeros32, initγ = Flux.ones32,
-                               ϵ = 1.0f-5, momentum = 1.0f-1,
+function depthwise_sep_conv_bn(kernel_size, inplanes, outplanes, activation = relu;
+                               prenorm = false, use_bn = (true, true),
```

Comment:
If you aren't going to allow the norm layer to be customized here, then you should keep the batch norm keywords and pass them into `conv_norm` via a closure over `BatchNorm`.

Reply:
Oh this has to be done alright, I just forgot it in all the churn going on in this PR 😅